docs/en/reference/Metrics-List.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: global
+title: List of Metrics
+group: Reference
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+There are two types of metrics in Alluxio, cluster-wide aggregated metrics, and per-process detailed metrics.
+
+Cluster metrics are collected and calculated by the leading master and displayed in the metrics tab of the web UI. 
+These metrics are designed to provide a snapshot of the cluster state and the overall amount of data and metadata served by Alluxio.
+
+Process metrics are collected by each Alluxio process and exposed in a machine-readable format through any configured sinks. 
+Process metrics are highly detailed and are intended to be consumed by third-party monitoring tools. 
+Users can then view fine-grained dashboards with time-series graphs of each metric, 
+such as data transferred or the number of RPC invocations.
+
+Metrics in Alluxio have the following format for master node metrics:
+
+```
+Master.[metricName].[tag1].[tag2]...
+```
+
+Metrics in Alluxio have the following format for non-master node metrics:
+
+```
+[processType].[metricName].[tag1].[tag2]...[hostName]
+```
+
+There is generally an Alluxio metric for every RPC invocation, to Alluxio or to the under store.
+
+Tags are additional pieces of metadata for the metric such as user name or under storage location.
+Tags can be used to further filter or aggregate on various characteristics.
+
+## Cluster Metrics
+
+Workers and clients send metrics data to the Alluxio master through heartbeats.
+The interval is defined by property `alluxio.master.worker.heartbeat.interval` and `alluxio.user.metrics.heartbeat.interval` respectively.
+
+<table class="table table-striped">
+<tr><th>Name</th><th>Type</th><th>Description</th></tr>
+{% for item in site.data.table.cluster-metrics %}
+  <tr>
+    <td><a class="anchor" name="{{ item.metricName }}"></a> {{ item.metricName }}</td>
+    <td>{{ item.metricType }}</td>
+    <td>{{ site.data.table.en.cluster-metrics[item.metricName] }}</td>
+  </tr>
+{% endfor %}
+</table>
+
+Bytes metrics are aggregated value from workers or clients. Bytes throughput metrics are calculated on the leading master.
+The values of bytes throughput metrics equal to bytes metrics counter value divided by the metrics record time and shown as bytes per minute.
```

Comment:
I think it would be better if we moved this above the table. Most people probably won't read after the end of the table

Reply:
Make sense, move it to before the table